cubedash/templates/layout/macros.html

```diff
@@ -52,7 +52,7 @@
 {% macro query_param_list(d, prefix="", wide=false, show_nulls=true, show_dicts=true, descriptions=None, fallback_dict={}) -%}
     <div class="query-param-list {% if wide %}wide{% endif %}">
         {% for key, value in d | dictsort if (show_dicts or value is not mapping) recursive %}
-            {% if show_nulls or (value is not none) or (key in fallback_dict) %}
+            {% if (show_nulls or (value is not none) or (key in fallback_dict)) and (key != "creation_dt") %}
```

Comment:
Not essential, but I do like to leave comments on lines like this, as it will be a head-scratcher for future programmers who see it.

```suggestion
            {# "creation_dt" is excluded, as it's a duplicate of the creation_time field (latter is newer and date typed) #}
            {% if (show_nulls or (value is not none) or (key in fallback_dict)) and (key != "creation_dt") %}
```